frontend/javascripts/admin/time/time_line_view.js

```diff
@@ -88,14 +105,19 @@ class TimeLineView extends React.PureComponent<*, State> {
     });
   }
 
+  fetchDataFromLoggedInUser = async () => {
+    await this.setState({ user: this.props.activeUser });
+    this.fetchTimeTrackingData();
+  };
+
   handleUserChange = async (userId: number) => {
     await this.setState(prevState => ({ user: prevState.users.find(u => u.id === userId) }));
     this.fetchTimeTrackingData();
   };
 
   handleDateChange = async (dates: DateRange) => {
     // to ease the load on the server restrict date range selection to a month
-    if (dates[0].diff(dates[1], "days") > 31) {
+    if (Math.abs(dates[0].diff(dates[1], "days")) > 31) {
```

Comment:
Before it was possible to avoid this warning. With the absolute value, this is now solved.